cloud-config/cloud-config-base.yml

```diff
@@ -15,6 +15,17 @@ vm_types:
       type: gp2
       size: 30000
       encrypted: true
+- name: errand_small
+  cloud_properties:
+    instance_type: t2.micro
```

Comment:
t2.small?  We probably need at least 2GB of ram with all the addons we pack onto these machines